controller/controller_test.go

```diff
@@ -462,16 +465,16 @@ func TestWhenMultipleControllerConsidersAllFilteredComain(t *testing.T) {
 						DNSName:    "some-record.used.tld",
 						RecordType: endpoint.RecordTypeA,
 						Targets:    endpoint.Targets{"8.8.8.8"},
-						Labels:     endpoint.Labels{},
 					},
 				},
 				UpdateNew: []*endpoint.Endpoint{
 					{
 						DNSName:    "some-record.used.tld",
 						RecordType: endpoint.RecordTypeA,
 						Targets:    endpoint.Targets{"1.1.1.1"},
+						// this ne new resolver will transfer existing labels
```

Comment:
The existing code copies only the `owner` label. If adding an assertion for that, then the test case should also include a second label and assert that the code does not (or does) copy that.